packages/babel-plugin/src/css-prop/__tests__/behaviour.test.ts

```diff
@@ -293,7 +293,7 @@ describe('css prop behaviour', () => {
     `);
 
     expect(actual).toInclude(
-      `{background-image:linear-gradient(45deg,gray 25%,transparent 25%),linear-gradient(-45deg,gray 25%,transparent 25%),linear-gradient(45deg,transparent 75%,gray 75%),linear-gradient(-45deg,transparent 75%,gray 75%)}`
+      `{background-image:linear-gradient(45deg,gray 25%,transparent 0),linear-gradient(-45deg,gray 25%,transparent 0),linear-gradient(45deg,transparent 75%,gray 0),linear-gradient(-45deg,transparent 75%,gray 0)}`
```

Comment:
how comes 25% becomes 0 here?

Reply:
just saw your comment. so 

linear-gradient(45deg,gray 25%,transparent 25%) = linear-gradient(45deg,gray 25%,transparent 0)?

Reply:
Yeahhhh, I had the same reaction too and[ visually checked ](https://github.com/atlassian-labs/compiled/pull/1266#discussion_r938487726). It is handled by https://github.com/atlassian-labs/compiled/blob/356b120030ffdd36f1872ad62efe163fc309e0f2/packages/css/src/plugins/normalize-css.ts#L28 

haven't checked the plugin, but [here](https://github.com/cssnano/cssnano/tree/master/packages/postcss-minify-gradients) the docs if you want to know more